commcare_connect/users/views.py

```diff
@@ -161,3 +161,19 @@ def post(self, *args, **kwargs):
                 user_invite.status = UserInviteStatus.sms_not_delivered
             user_invite.save()
         return Response(status=200)
+
+
+@csrf_exempt
+@api_view(["POST"])
+@authentication_classes([OAuth2Authentication])
+def user_deactivate(request):
```

Comment:
is mobile supposed to call this or does connectid do it?

Reply:
Mobile wil call this. I spot the problem here. Thanks. : )

Reply:
This is called by mobile.

Reply:
I think it would be safer and better if this were called by connectid. That way we don't need to rely on mobile, and as soon as a user account is deactivated we can be sure no new data will flow in

Reply:
Yeah, that makes sense. We need to think about the Authentication between Connect and ConnectID for making requests to this API.

Reply:
Sravan made a good point against this on the spec. Let's chat about this offline and make a decision